applications/nrf5340_audio/README.rst

```diff
@@ -1344,6 +1344,13 @@ To use the nRF5340 Audio application with your custom board:
 #. Build the application by selecting the name of the new board (for example, ``new_audio_board_name``) in your build system.
    For example, when building from the command line, add ``-b new_audio_board_name`` to your build command.
 
+FOTA for end products
+=====================
+
+The default MCUBoot key should not be used for end products. See :ref:`west-sign` for more information.
```

Comment:
```suggestion
Do not use the default MCUBoot key for end products.
See :ref:`west-sign` for more information.
```

Also isn't a link to ug_fw_update more appropriate? https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_fw_update.html

Reply:
Done. Changed to this link :)

Reply:
Edit: I added both links, since it looks like they contain some different information